README.md

```diff
@@ -503,6 +503,79 @@ You will need an existing ProjectReference and LodgementId in order to do this.
 | - workCarriedOutByTMLN            | The TMLN of the registered business that carried out the work |
 | - workCarriedOutBySchemeId        | The TrustMark Scheme Provider schemeId for the installer of this work |
 
+#### StandaloneLodgementSubmit
+
+> POST /Data/StandaloneLodgementSubmit
+
+Creates a project and lodgement submission in a single API call.
```

Comment:
This supports the lodgement of work conducted under TrustMark Licence Plus.

Reply:
Is there any need to mention the concept of a project here @phil-vaughan?